Automattic/jetpack
# [WIP] Hide the 'share this' section in the print post view.

Fixes https://github.com/Automattic/jetpack/issues/16205

Jetpack should automatically hide extraneous elements like Likes, Sharing, and Related posts in print media. These are interactive elements and don't make sense in the context of a printed page.

In this PR, we are going to hide the "share this" section in the "print" view. 

I have added some screenshots to explain this. Consider the post below:

<img width="1119" alt="Screenshot 2021-11-30 at 4 03 30 PM" src="https://user-images.githubusercontent.com/6594561/144054770-9e8b7c22-2e6a-4e07-85c2-455e14deb399.png">

If one were to print this view, this is how it would look. 

<img width="627" alt="Screenshot 2021-11-30 at 4 04 41 PM" src="https://user-images.githubusercontent.com/6594561/144054843-90ab204f-2a8f-43e5-8036-41ee9f42dcc7.png">

How we want it to look (this PR makes the changes necessary to make the page look like the image below): 

<img width="653" alt="Screenshot 2021-11-30 at 4 05 56 PM" src="https://user-images.githubusercontent.com/6594561/144054914-04071e04-8cbe-4cf3-b2ca-8dbdd8a84773.png">


#### Changes proposed in this Pull Request:

* Added a CSS media query that hides the CSS class `sharedaddy.sd-sharing-enabled` in the print view.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
 - Go to Jetpack->Settings->Sharing.
 - Make sure "Add sharing buttons to your posts and pages" is enabled
 - Create a post, go to its detailed view and print it (you can press cmd+p to print on a Mac)
 - Observe the results and make sure that the "share this" section is not showing up in the print view
 -  Go to Jetpack->Settings->Sharing, click on "Configure Sharing buttons"
 -  Click on edit label text and change the text to whatever you like. 
 -  Change the button style. 
 -  Go to the post detail view again and confirm that the "share this" section doesn't show up.
